pkg/kubectl/cmd/annotate.go

```diff
@@ -190,10 +190,12 @@ func (o AnnotateOptions) RunAnnotate(f cmdutil.Factory, cmd *cobra.Command) erro
 		return err
 	}
 
+	includeUninitialized := cmdutil.GetFlagBool(cmd, "include-uninitialized")
```

Comment:
See my comment at  https://github.com/kubernetes/kubernetes/issues/49035#issuecomment-322322225 I think `annotate` works with individual resources (and doesn't work in batch mode) so it should not have this flag.